tests/Feature/Auth/RegistrationTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\Auth;
+
+use Livewire\Volt\Volt;
+
+test('registration screen can be rendered', function (): void {
+    $response = $this->get('/register');
+
+    $response
+        ->assertOk()
+        ->assertSeeVolt('pages.auth.register');
+});
+
+test('new users can register', function (): void {
+    $component = Volt::test('pages.auth.register')
+        ->set('name', 'Test User')
+        ->set('email', 'test@example.com')
+        ->set('password', 'password');
+
+    $component->call('register');
+
+    $this->assertGuest();
+
+    $this->assertSessionHas('status', __('pages/auth.register.email_verification_status'));
+
+    $this->assertAuthenticated();
+});
```

Comment:
assure toi qu'il toi de vérifier aussi que les champs sont requis. avec un test `fields must be required`
pour ton second test le nom doit être new `new users can be registered`

Reply:
Je ne comprends pas ce que tu veux dire